### PR TITLE
Rerender place highlighting not clean

### PR DESCRIPTION
**Pull request by Trufi**
Steps to reproduce:
1.Open http://maps.api.2gis.ru/2.0/
2. Click in district (on start zoom level)
3. Execute in console
```js
['en', 'es'].forEach(function(lang){ map.setLang(lang)})
```
4.Close callout
Expected: clean map with marker
But got: map with marker and hightlighted place
Screenshot:
screenshot from 2015-04-21 12 59 53

### CODE REVIEW

**Comment by ilfa** [comment]
:+1:

File: src/DGGeoclicker/src/handler/CityArea.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ DG.Geoclicker.Handler.CityArea = DG.Geoclicker.Handler.Default.extend({
             this._initStyles();
         }
 
+        // remove previously geometry see https://github.com/2gis/mapsapi/issues/213
```
Comment (RLRR) [inline src/DGGeoclicker/src/handler/CityArea.js]: Лучше исправить на «previous»)

Comment (Trufi) [inline src/DGGeoclicker/src/handler/CityArea.js]: Cпасибо) Поправил

**Comment by RLRR** [comment]
:+1:

**Pull request merged**